### PR TITLE
RelatedApps | New Icons

### PR DESCRIPTION
**Pull request by ZilvinasMaciulis**

### CODE REVIEW

File: packages/wix-ui-icons-common/src/general/metadata.ts [hunk 1]
```diff
@@ -648,6 +648,13 @@ sizes: { "18": "GetStartedSmall", "24": "GetStarted" },
     tags: ["globe"],
     sizes: { "18": "PublicSmall", "24": "Public" },
   },
+{
+  title:“RelatedApps”,
```
Comment (mykas) [inline packages/wix-ui-icons-common/src/general/metadata.ts:652]:
```suggestion
  title: "RelatedApps",
```

Comment (Zivilekin) [reply packages/wix-ui-icons-common/src/general/metadata.ts:652]: will this be accepted? @ZilvinasMaciulis

Comment (ZilvinasMaciulis) [reply packages/wix-ui-icons-common/src/general/metadata.ts]: @Zivilekin what do you mean?

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-icons-common**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-icons-common.pr_2083/

**Review by Zivilekin — APPROVED** [review]

**Pull request merged**